framework/src/controller/network/network.js

```diff
@@ -48,46 +45,15 @@ const NETWORK_INFO_KEY_TRIED_PEERS = 'tried_peers_list';
 const DEFAULT_PEER_SAVE_INTERVAL = 10 * 60 * 1000; // 10min in ms
 
 module.exports = class Network {
-	constructor(options) {
-		this.options = options;
-		this.channel = null;
-		this.logger = null;
-		this.storage = null;
+	constructor({ networkConfig, channel, logger, storage }) {
+		this.networkConfig = networkConfig;
```

Comment:
What's the need to change `options` to `networkConfig`? I would be rather consistent to use `options` in ever class.